Makefile

```
@@ -242,18 +249,18 @@ install-deps-fedora:
 lint:
 	. $(VENVNAME)/bin/activate; \
 	echo "--- Linting ... ---" && \
-	SEARCH_PATH=$(REPOS_PATH) && \
+	SEARCH_PATH="$(TEST_PATHS)" && \
 	echo "Using search path '$${SEARCH_PATH}'" && \
 	echo "--- Running pylint ---" && \
-	bash -c "[[ ! -z $${SEARCH_PATH} ]] && find $${SEARCH_PATH} -name '*.py' | sort -u | xargs pylint" && \
+	bash -c "[[ ! -z '$${SEARCH_PATH}' ]] && find $${SEARCH_PATH} -name '*.py' | sort -u | xargs pylint -j 8" && \
```

@vinzenz  what about to specify rather `0`, so pylint itself discover the number of processors that can be used? Seems to me better for machines that have just 4 processors.
```suggestion
	bash -c "[[ ! -z '$${SEARCH_PATH}' ]] && find $${SEARCH_PATH} -name '*.py' | sort -u | xargs pylint -j 0" && \
```